projects/pyqt.cmake

```diff
@@ -1,7 +1,7 @@
 set(pyqt_command "")
 if(BUILD_OS_WINDOWS)
     set(pyqt_command
-        ${PYTHON_EXECUTABLE} configure.py
+        ${PYTHON_EXECUTABLE_PREFIXED} configure.py
```

Comment:
Ok, but I still saw in our internal docs that is was needed somehow, but I'll retry.